domain/network/state/space.go

```diff
@@ -125,7 +125,11 @@ WHERE  uuid = $Space.uuid;`, SpaceSubnetRow{}, space)
 		return nil, errors.Annotate(err, "querying spaces")
 	}
 
-	return &spaceRows.ToSpaceInfos()[0], nil
+	infos, err := spaceRows.ToSpaceInfos()
+	if err != nil {
+		return nil, errors.Trace(err)
```

Comment:
Not using `errors.Trace` any more.

Reply:
I didn't want to change just one place in a package which uses Trace all over. I'd rather do a cleanup PR per package where everything is fixed together.

Reply:
Forgot to mention, this was also discussed with Tom and we're not ready yet as there's no agreed replacement.